packages/from-to-location-picker/src/index.tsx

```diff
@@ -0,0 +1,81 @@
+import flatten from "flat";
+import LocationIcon from "@opentripplanner/location-icon";
+import { FormattedMessage } from "react-intl";
+import React from "react";
+// eslint-disable-next-line prettier/prettier
+import type { FromToPickerProps } from "./types";
```

Comment:
Move `import type` between `import styled` and `import defaultEnglishMessages`.

Reply:
Good note thank you! Reordered in da90c95eb425b586d677d17eb3885b216bb45271